include/circt/Dialect/FIRRTL/FIRRTLIntrinsics.td

```diff
@@ -16,23 +16,35 @@
 include "FIRRTLDialect.td"
 
 //===----------------------------------------------------------------------===//
-// Generic intrinsic operation for parsing into before lowering.
+// Generic intrinsic operations for parsing into before lowering.
 //===----------------------------------------------------------------------===//
 
-def GenericIntrinsicOp : FIRRTLOp<"int.generic",
+def GenericIntrinsicExprOp : FIRRTLOp<"int.generic.expr",
     [HasCustomSSAName]> {
```

Comment:
Is it necessary to mark this pure if we use it as an expression?

Reply:
I must've lost that in my shuffling/rebasing, good catch thank you.